src/brain/gocdConsecutiveUnsuccessfulAlert/index.test.ts

```diff
@@ -0,0 +1,235 @@
+import merge from 'lodash.merge';
+
+import payload from '@test/payloads/gocd/gocd-stage-building.json';
+
+import * as slackblocks from '@/blocks/slackBlocks';
+import { DB_TABLE_STAGES } from '@/brain/saveGoCDStageEvents';
+import { buildServer } from '@/buildServer';
+import {
+  FEED_DEV_INFRA_CHANNEL_ID,
+  GOCD_SENTRYIO_BE_PIPELINE_GROUP,
+  GOCD_SENTRYIO_BE_PIPELINE_NAME,
+} from '@/config';
+import { Fastify } from '@/types';
+import { bolt } from '@api/slack';
+import { db } from '@utils/db';
+
+import { CONSECUTIVE_UNSUCCESSFUL_DEPLOYS_LIMIT } from './consecutiveUnsuccessfulDeploysAlert';
+import { gocdConsecutiveUnsuccessfulAlert, handler } from '.';
+
+jest.mock('@api/getUser');
+
+describe('gocdConsecutiveUnsuccessfulAlerts', function () {
+  let fastify: Fastify;
+
+  beforeAll(async function () {
+    await db.migrate.latest();
+  });
+
+  afterAll(async function () {
+    await db.destroy();
+  });
+
+  beforeEach(async function () {
+    fastify = await buildServer(false);
+    await gocdConsecutiveUnsuccessfulAlert();
+    await db(DB_TABLE_STAGES).delete();
```

Comment:
Is it expected to 
```
await db(DB_TABLE_STAGES).delete();
await db('slack_messages').delete();
```

on `beforeEach` and `afterEach`?

Reply:
I just followed the pattern used in https://github.com/getsentry/eng-pipes/blob/main/src/brain/gocdSlackFeeds/deployFeed.test.ts